data/styles/clear/include/Roads.mapcss

```diff
@@ -659,31 +667,17 @@ line|z16-[highway=pedestrian][tunnel?]
 line|z13[highway=pedestrian][bridge?]::bridgewhite,
 {casing-width-add: -0.2;}
 line|z14-[highway=pedestrian][bridge?]::bridgewhite,
-/* TODO: test this style before adding
-line|z15-[highway=footway][bridge?]::bridgewhite,
-line|z15-[highway=steps][bridge?]::bridgewhite,
-*/
 line|z16-[highway=road][bridge?]::bridgewhite,
 line|z16-[highway=service][bridge?]::bridgewhite
 {casing-width-add: 0;}
-line|z16-[highway=pedestrian][bridge?]::bridgewhite,
-/* TODO: test this style before adding
-line|z16-[highway=footway][bridge?]::bridgewhite
-*/
+line|z16-[highway=pedestrian][bridge?]::bridgewhite
 {casing-width-add: 0.2;}
 
 line|z14-[highway=pedestrian][bridge?]::bridgeblack,
-/* TODO: test this style before adding
-line|z15-[highway=footway][bridge?]::bridgeblack,
-line|z15-[highway=steps][bridge?]::bridgeblack,
-*/
 line|z16-[highway=road][bridge?]::bridgeblack,
 line|z16-[highway=service][bridge?]::bridgeblack
 {casing-width-add: 0.2;}
-line|z16-[highway=pedestrian][bridge?]::bridgeblack,
-/* TODO: test this style before adding
-line|z16-[highway=footway][bridge?]::bridgeblack,
-*/
+line|z16-[highway=pedestrian][bridge?]::bridgeblack
```

Comment:
nit: its better to keep a trailing comma everywhere, this way its less diff noise with subsequent additions

Reply:
resolved